cfe_internal/core/watchdog/templates/watchdog.mustache

```diff
@@ -273,7 +273,7 @@ if [ "${PATHOLOGY_COUNT}" -gt "${PATHOLOGY_THRESHOLD}" ]; then
 
     # Switch to more intelligent cf-check repair in 3.12.3
     echo "- Purged LMDBs" >> ${COLLECTION_REPORT}
-    for each in $(ls ${CFENGINE_WORKDIR}/state/*.lmdb*); do
+    for each in "${CFENGINE_WORKDIR}/state/*.lmdb*"; do
```

Comment:
The `*`s have to be outside of the double-quotes otherwise they won't expand.

Reply:
ack